spatial-lib/src/cmr/spatial/points_validation_helpers.clj

```diff
@@ -20,7 +45,7 @@
   map of rounded points to list of index, point pairs."
   [points]
   (reduce (fn [m [i point]]
-            (let [rounded (p/round-point 8 point)]
+            (let [rounded (p/round-point elasticsearch-rounding-precision point)]
```

Comment:
This syncs what CMR was doing and what ES could handle after converting ordinates to integer and back.